src/components/composite/layout/TopBar/SearchBar.tsx

```diff
@@ -78,6 +79,9 @@ const SearchBar = () => {
             <TopSearchComponent />
           </Box>
         </Hidden>
+        <Box>
```

Comment:
Please apply a small margin at least to the right to look better on smaller screens.